airbyte-webapp/src/config/defaultConfig.ts

```diff
@@ -5,6 +5,7 @@ const defaultConfig: Config = {
   healthCheckInterval: 20000,
   version: "dev",
   apiUrl: `${window.location.protocol}//${window.location.hostname}:8001/api`,
+  connectorBuilderApiUrl: `${window.location.protocol}//${window.location.hostname}:8080/`,
```

Comment:
8080 seems like a very common port

Reply:
Happy to change it to whatever port we think is appropriate

Reply:
We could use 8003, since it seems like that is not currently used by anything else in OSS or Cloud (8001 is used by airbyte-server, and 8002 is used by the airyte-cloud-server in Cloud)

Reply:
Either way, I'm just going to commit this as is for now, since we can easily change this later